app/routes/umd_tree_cover_loss/raster_tiles.py

```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
     background_tasks: BackgroundTasks,
 ) -> Response:
     """
-    UMD tree cover loss raster tile.
+    UMD Tree Cover Loss raster tile.
+
+    There are two types of TCL tiles available here: True color tiles and encoded tiles.
+    The true color tiles have been pre-rendered to appear the same way TCL appears on
+    globalforestwatch.org (i.e. the tiles show pink pixels to indicate tree cover loss).
+    The caveat with the true color tiles is that ANY tree cover loss over the history of
+    the data is shown as a pink pixel. There's no way to filter out pixels which fall in
+    or out of a range of years like there is in the map interface, because that
+    information is no longer in the data!
```

Comment:
Lines 64-67 are a bit confusing.  I would actually say something more like:

The caveat is that tile data fetched by a single tile URL is only for a specific range of years (as specified below via query parameters).  The tile directly displays the forest loss for that range of years as pink pixels.  So, the true color tiles cannot be used with a dynamic slider to adjust the year range.  To allow for a dynamically adjustable year range, you should use the encoded tiles.